tests/performance_tests/performance_test_basic.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, testHelperConfig: TestHelperConfig=TestHelperConfig(), cluste
 
         self.rootLogDir = rootLogDir
         self.ptbLogDir = f"{self.rootLogDir}/{os.path.splitext(os.path.basename(__file__))[0]}"
-        self.testTimeStampDirPath = f"{self.ptbLogDir}/{datetime.now().strftime('%Y-%m-%d_%H-%M-%S')}"
+        self.testTimeStampDirPath = f"{self.ptbLogDir}/{datetime.utcnow().strftime('%Y-%m-%d_%H-%M-%S')}"
```

Comment:
For traceability, shouldn't this be using `self.testStart` rather than whatever time it happens to be when this executes?  That would require initializing `self.testStart` here in `__init__` rather than in `runTest()` but the difference should be negligible as the very next thing `main()` does after construction is execute `runTest()`.  Having the directory timestamp precisely match the logged timestamp will help developers tracking things down in the future and avoid difficulties in further machine processing down the road (from historical archives or some such).

Reply:
Good call.  Addressed: [5ffdb64](https://github.com/AntelopeIO/leap/pull/360/commits/5ffdb64ab116ce895fc89395271e96aae7eda7d7)

Reply:
Did the same for `performance_test.py`: [34e5857](https://github.com/AntelopeIO/leap/pull/360/commits/34e5857866fae97e8800720defc53b0f6b652639)